engine/pom.xml

```
@@ -260,7 +260,7 @@
                         <Bundle-SymbolicName>${hibernate-validator.module-name}</Bundle-SymbolicName>
                         <Import-Package>
                             jakarta.persistence.*;version="[3.0.0,4.0.0)";resolution:=optional,
-                            jakarta.validation.*;version="[3.0.0,4.0.0)",
+                            jakarta.validation.*;version="[${version.jakarta.validation-api},4.0.0)",
```

If you parse versions... may as well take advantage of `nextMajorVersion`. See https://www.mojohaus.org/build-helper-maven-plugin/parse-version-mojo.html

```suggestion
                            jakarta.validation.*;version="[${version.jakarta.validation-api},${parsed-version.jakarta.validation-api.nextMajorVersion}.0.0)",
```

---

oh nice 👍🏻